scripts/run_goreleaser.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+set -x
+[[ -z ${DEBUG:-} ]] || set -o xtrace
+
+DIR="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && cd .. && pwd -P)"
```

Comment:
can you add a comment for what this is doing, I think it's cd'ing to the directory of the bash script, then cd'ing up from there and printing the dir?